### DISCUSSION TITLE
Camera trap data mobilisation

### DISCUSSION THREAD

**Discussion by MichalTorma** [gbif-norway/helpdesk#137] [General]
We had a node meeting on 8.3.2023 about camera trap data mobilisation. To continue the discussion around this topic, I wanted to share this article written by Artsdatabanke, GBIF and collaborators: https://www.researchgate.net/publication/292983750_Publishing_Camera_Trap_Data_a_Best_Practice_Guide What are your thoughts on the best practices outlined in this article?

**Comment by dagendresen** [comment]
Recently (September 2021) there was a [GBIF call for a small project funding](https://www.gbif.org/news/35nbYlsHyz9NWIFS2ySoEI/call-for-author-proposals-for-new-and-updated-gbif-technical-guides-closed) for writing a best practice technical guideline (on collection digitization, `publishing camera-trap data`, filling data gaps and reducing biases, and species migration and telemetry data). LivingNorway and NINA submitted a (self-funded) proposal for the camera-trap guideline. Anyway, we might see an update to the [2017 camera-trap guideline](https://www.gbif.org/document/1o6HNHuCxKaiAC8yG86gQq/publishing-camera-trap-data-a-best-practice-guide) soon.

As part of the work on the [new GBIF data model](https://www.gbif.org/new-data-model), the [third webinar](https://www.gbif.org/event/f68927-b5c1-4ac8-a4ac-7d47645/exploring-camera-trap-data) was about the [camera-trap data use case](https://docs.google.com/document/d/1DkrjaAyabHc1zqvcJxivTiHp45jQOmGfE4beAtVsUi8/edit). This is an on-going activity at the GBIF Secretariat with very good chances for actual uptake of suggestions made to the use case!
